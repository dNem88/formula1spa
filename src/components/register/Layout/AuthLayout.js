import React from 'react'
import {Outlet} from 'react-router-dom'
import SelectForm from '../Select/SelectForm'
import styles from './authLayout.module.css'
function AuthLayout() {
    return (
        <div className={styles.container}>
            <SelectForm location={'auth'}/>
            <div className={styles['form-container']}>
                <Outlet/>
            </div>
        </div>
    )
}

export default AuthLayout