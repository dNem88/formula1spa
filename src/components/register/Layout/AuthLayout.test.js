import renderer from 'react-test-renderer';
import AuthLayout from './AuthLayout'
import React from 'react'
import {
    MemoryRouter as Router
} from 'react-router-dom'
describe('test  AuthLayout component', () => {
  it('Should render Layout for login and register forms', () => {
    let layout = renderer.create(
        <Router>
             <AuthLayout/>
        </Router>
    ).toJSON()
    expect(layout).toMatchSnapshot()
    expect(layout.children.length).toBe(2)
    expect(layout.children[0].type).toBe('div')
    expect(layout.children[1].type).toBe('div')
  })
  
})