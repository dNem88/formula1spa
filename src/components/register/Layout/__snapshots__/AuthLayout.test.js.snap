// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test  AuthLayout component Should render Layout for login and register forms 1`] = `
<div
  className="container"
>
  <div
    className="container"
  >
    <a
      className=""
      href="/auth/login"
      onClick={[Function]}
    >
      Sign in
    </a>
    <a
      className=""
      href="/auth/register"
      onClick={[Function]}
    >
      Register
    </a>
  </div>
  <div
    className="form-container"
  />
</div>
`;
